.ci/download_image.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+components=(
+    zookeeper
+    zookeeper.customTools.backup
+    zookeeper.customTools.restore
+    bookie
+    presto
+    autorecovery
+    broker
+    proxy
+    pulsar_detector
+    functions
+    function_worker
+    toolset
+    toolset.kafka
+    prometheus
+    alert_manager
+    grafana
+    streamnative_console
+    node_exporter
+    nginx_ingress_controller
+    vault
+    vault_init
+    custom_metric_server_prometheus
+    custom_metric_server
+    pulsar_metadata
+    configmapReload
+    external_dns)
+sn_platform_tag=$1
```

Comment:
I think you should (default this)[https://stackoverflow.com/a/2013589] or (throw a help message & exit)[https://stackoverflow.com/a/8968775] if not provided.